src/classes/Qonversion.ts

```diff
@@ -20,7 +20,7 @@ export default class Qonversion {
     key: string,
     observerMode: boolean = false
   ): Promise<LaunchResult> {
-    RNQonversion.storeSDKInfo(sourceKey, "rn", versionKey, sdkVersion);
+    await RNQonversion.storeSDKInfo(sourceKey, "rn", versionKey, sdkVersion);
```

Comment:
As we discussed above, `await` should be removed because `storeSDKInfo` function no need to have a promise

Reply:
Done